Sources/SimpleNetworkService/SimpleNetworkService.swift

```diff
@@ -0,0 +1,8 @@
+/// A stub object for future use as a network layer.
+final public class SimpleNetworkService {
```

Comment:
Why don't you give just the `NetworkService` name?

Remember: You don't have to name your classes as your package.

Reply:
fixed